source/isaaclab/isaaclab/sim/spawners/meshes/meshes.py

```diff
@@ -230,6 +230,43 @@ def spawn_mesh_cone(
     return prim_utils.get_prim_at_path(prim_path)
 
 
+@clone
```

Comment:
Hm from original design, this spawning function should be inside the `from_files.py` sub-module. Could you please move it there?